extractor/fetch_index.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env sh
+
+set -e
+
+STATE=mt
+# TODO: Allow multiple years
+YEAR=2019
+
+# TODO: Convert to a makefile to avoid redundant downloads
+# TODO: Check for irsx_index in the PATH
```

Comment:
For posterity: https://github.com/jsfenfen/990-xml-reader/blob/master/README.md#irsx_index